apps/freeablo/fagui/shopdialogs.h

```diff
@@ -47,10 +69,10 @@ namespace FAGui
         virtual DialogData getDialogData() override;
 
     private:
-        void buyItem(size_t index);
+        void buyItem(size_t index, ConfirmTransactionPopup* confirmPopup);
 
     private:
         std::vector<FAWorld::StoreItem>& mItems;
         const FAWorld::Actor& mShopkeeper;
     };
-}
+} // namespace FAGui
```

Comment:
Nitpick, but I'm not a fan of these end namespace comments. I presume they were added by your IDE?

Reply:
It's defined in your .clang-format, see FixNamespaceComments in https://clang.llvm.org/docs/ClangFormatStyleOptions.html
Apply preference here:
https://github.com/wheybags/freeablo/blob/master/.clang-format

Reply:
From what I can tell, we want that option off, and it should be off by default since we didn't enable it? Unless I'm missing something?

Reply:
Afaik it's on by default

Reply:
Well, either way, I just set it to false explicitly now.

Reply:
It was added by Visual Studio, sorry about that.

Reply:
It was added by Visual Studio, sorry about that.

Reply:
Visual Studio **Code** uses clang-format afaik.